sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -558,11 +561,19 @@ pkg_postinst() {
 	# Flatcar: Reenabling systemd-timesyncd service too.
 	systemd_reenable systemd-networkd.service systemd-resolved.service systemd-timesyncd.service
 
+	# Flatcar: TODO: Possibly replace `systemctl enable` with
```

Comment:
Looking at `portage-stable/eclass/systemd.eclass` where `systemd_reenable` is defined I guess we can remove the `ENABLED_UNITS` usage and the `save_enabled_units` call for it when we solely use `systemd_reenable`.

Reply:
So, I think we shouldn't be using this at all, since they all put stuff into `/etc`.

Reply:
See comment above, it's correctly working.

Reply:
Or do you mean not using `ENABLED_UNITS`?

Reply:
This got dropped.